arm9/modules/12/asm/module_12.s

```diff
@@ -48104,64 +48104,176 @@ _02244988:
 	.byte 0x00, 0x00, 0x00, 0x00, 0x01, 0x00, 0x0F, 0x00, 0x00, 0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+
+	.global MOD12_02244AFC
+MOD12_02244AFC: ; 0x02244AFC
 	.byte 0x00, 0x00, 0x00, 0x00, 0x80, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x20, 0x00, 0x00, 0x00
 	.byte 0x00, 0x00, 0x00, 0x00, 0x80, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x20, 0x00, 0x00, 0x00
+
+	.global MOD12_02244B1C
+MOD12_02244B1C: ; 0x02244B1C
 	.byte 0x04, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x80, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
 	.byte 0x10, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
-	.byte 0x03, 0x00, 0x00, 0x00, 0x60, 0x00, 0x00, 0x00, 0x91, 0xF9, 0x22, 0x02, 0xC5, 0xF9, 0x22, 0x02
-	.byte 0x3D, 0xFA, 0x22, 0x02, 0xA1, 0xFA, 0x22, 0x02, 0xB1, 0xFB, 0x22, 0x02, 0x61, 0xFB, 0x22, 0x02
-	.byte 0x11, 0xFB, 0x22, 0x02, 0x61, 0xFC, 0x22, 0x02, 0xF1, 0xFC, 0x22, 0x02, 0x21, 0xFD, 0x22, 0x02
-	.byte 0xFD, 0xFD, 0x22, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x10, 0x00, 0x00
+	.byte 0x03, 0x00, 0x00, 0x00, 0x60, 0x00, 0x00, 0x00
+
+	.global MOD12_02244B44
+MOD12_02244B44: ; 0x02244B44
+	.word MOD12_0222F990, MOD12_0222F9C4
+	.word MOD12_0222FA3C, MOD12_0222FAA0, MOD12_0222FBB0, MOD12_0222FB60
+	.word MOD12_0222FB10, MOD12_0222FC60, MOD12_0222FCF0, MOD12_0222FD20
+	.word MOD12_0222FDFC
+
+	.global MOD12_02244B70
+MOD12_02244B70: ; 0x02244B70
+	.byte 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x10, 0x00, 0x00
 	.byte 0x00, 0x00, 0x00, 0x00, 0x03, 0x00, 0x00, 0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
 	.byte 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
 	.byte 0x04, 0x00, 0x02, 0x05, 0x00, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
 	.byte 0x00, 0x00, 0x00, 0x00, 0x00, 0x10, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x03, 0x00, 0x06, 0x01
-	.byte 0x00, 0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
-	.byte 0x00, 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x60, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x00
-	.byte 0x00, 0x40, 0x00, 0x00, 0x10, 0x00, 0x10, 0x00, 0x10, 0x00, 0x00, 0x00, 0x99, 0x04, 0x78, 0x00
-	.byte 0x02, 0x03, 0x04, 0x04, 0x02, 0x02, 0x03, 0x03, 0x9B, 0x04, 0x78, 0x00, 0x02, 0x04, 0x04, 0x04
-	.byte 0x02, 0x02, 0x03, 0x03, 0x60, 0x00, 0x00, 0x00, 0x20, 0x00, 0x00, 0x00, 0x40, 0x00, 0x00, 0x00
-	.byte 0x40, 0x00, 0x00, 0x00, 0x08, 0x00, 0x00, 0x00, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+	.byte 0x00, 0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+
+	.global MOD12_02244BC4
+MOD12_02244BC4: ; 0x02244BC4
+	.byte 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+	.byte 0x00, 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00
+
+	.global MOD12_02244BD4
+MOD12_02244BD4: ; 0x02244BD4
+	.byte 0x60, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x00
+	.byte 0x00, 0x40, 0x00, 0x00, 0x10, 0x00, 0x10, 0x00, 0x10, 0x00, 0x00, 0x00
+
+	.global MOD12_02244BE8
```

Comment:
Probably a struct. Subsequent labels should be replaced with this plus a fixed offset.